perception_launch/launch/object_segmentation/ground_segmentation/ground_segmentation.launch.py

```diff
@@ -0,0 +1,303 @@
+            {
+                "input_frame": LaunchConfiguration("base_frame"),
+                "output_frame": LaunchConfiguration("base_frame"),
+                "min_x": -50.0,
+                "max_x": 100.0,
+                "min_y": -50.0,
+                "max_y": 50.0,
+                "min_z": vehicle_info["min_height_offset"],
+                "max_z": vehicle_info["max_height_offset"],
+                "negative": False,
+            }
+        ],
+        extra_arguments=[{"use_intra_process_comms": LaunchConfiguration("use_intra_process")}],
+    )
+
+    customized_ground_component = ComposableNode(
+        package="ground_segmentation",
+        plugin="ground_segmentation::ScanGroundFilterComponent",
+        name="scan_ground_filter",
+        remappings=[
+            ("input", "measurement_range_cropped/pointcloud"),
+            ("output", "concatenated/no_ground/pointcloud"),
+        ],
+        parameters=[
+            {
+                "global_slope_max_angle_deg": 10.0,
+                "local_slope_max_angle_deg": 30.0,
+                "split_points_distance_tolerance": 0.2,
+                "split_height_distance": 0.2,
+                "use_virtual_ground_point": False,
+            }
+        ],
+        extra_arguments=[{"use_intra_process_comms": LaunchConfiguration("use_intra_process")}],
+    )
+
+    ground_concat_topics = ["concatenated/no_ground/pointcloud"]
+    for lidar_name in pipeline_param["additional_lidar"]:
+        ground_concat_topics.extend([lidar_name + "/no_ground/pointcloud"])
+
+    ground_concat_component = ComposableNode(
+        package="pointcloud_preprocessor",
+        plugin="pointcloud_preprocessor::PointCloudConcatenateDataSynchronizerComponent",
+        name="concatenate_data_front",
+        remappings=[("output", "no_ground/oneshot/pointcloud")],
+        parameters=[
+            {
+                "input_topics": ground_concat_topics,
+                "output_frame": LaunchConfiguration("base_frame"),
+            }
+        ],
+        extra_arguments=[{"use_intra_process_comms": LaunchConfiguration("use_intra_process")}],
+    )
+
+    ground_component = ComposableNode(
+        package="ground_segmentation",
+        plugin="ground_segmentation::ScanGroundFilterComponent",
+        name="scan_ground_filter",
+        remappings=[
+            ("input", "measurement_range_cropped/pointcloud"),
+            ("output", "no_ground/oneshot/pointcloud"),
+        ],
+        parameters=[
+            {
+                "global_slope_max_angle_deg": 10.0,
+                "local_slope_max_angle_deg": 30.0,
+                "split_points_distance_tolerance": 0.2,
+                "split_height_distance": 0.2,
+                "use_virtual_ground_point": False,
+            }
+        ],
+        extra_arguments=[{"use_intra_process_comms": LaunchConfiguration("use_intra_process")}],
+    )
+
+    load_laserscan_to_occupancy_grid_map = IncludeLaunchDescription(
```

Comment:
please rename `load_laserscan_to_occupancy_grid_map` to `laserscan_to_occupancy_grid_map_loader`

Reply:
fix inhttps://github.com/tier4/AutowareArchitectureProposal_launcher/pull/117/commits/92d24c1db2184978dd4fb8c144bf4ab388d0cc68